store/tikv/unionstore/memdb.go

```diff
@@ -60,8 +59,8 @@ type MemDB struct {
 
 	entrySizeLimit  uint64
 	bufferSizeLimit uint64
-	count           int64
-	size            int64
+	count           int
+	size            int
```

Comment:
Is the `size` the number of total bytes? I just realize that if a transaction's size exceeds 4GB, it's size may not fit in a `int` value. It's out of the topic of this PR though.